library/core/src/time.rs

```diff
@@ -620,13 +620,12 @@ impl Duration {
     /// Basic usage:
     ///
     /// ```
-    /// #![feature(duration_abs_diff)]
     /// use std::time::Duration;
     ///
     /// assert_eq!(Duration::new(100, 0).abs_diff(Duration::new(80, 0)), Duration::new(20, 0));
     /// assert_eq!(Duration::new(100, 400_000_000).abs_diff(Duration::new(110, 0)), Duration::new(9, 600_000_000));
     /// ```
-    #[unstable(feature = "duration_abs_diff", issue = "117618")]
+    #[stable(feature = "duration_abs_diff", since = "CURRENT_RUSTC_VERSION")]
     #[must_use = "this returns the result of the operation, \
                   without modifying the original"]
     #[inline]
```

Comment:
should be entirely uncontroversial, but @rust-lang/wg-const-eval this will be const

Reply:
`rustc_allow_const_fn_unstable` is fishy. But I guess worst-case we can replace the `unwrap` by a custom `match` if we somehow need to preserve const-ness here and const_precise_live_drops does not work out...

Reply:
we've done worse crimes than inlining `Option::unwrap`, sounds fine :D

Reply:
> rustc_allow_const_fn_unstable is fishy.

I got that impression as well, I didn't expect having to add this. Should I get rid of this in favor of a match?

Reply:
I think it's ok to keep as-is, it's just a convenience that could be removed in the future if necessary, but we can keep while we can.